connector/Dockerfile

```diff
@@ -26,7 +26,7 @@ COPY --from=build /home/gradle/project/connector/build/libs/app.jar /app
 COPY ./connector/src/main/resources/logging.properties /app
 
 # health status is determined by the availability of the /health endpoint
-HEALTHCHECK --interval=5s --timeout=5s --retries=10 CMD curl -H "X-Api-Key: $EDC_API_AUTH_KEY" --fail http://localhost:8181/api/check/health
+# HEALTHCHECK --interval=5s --timeout=5s --retries=10 CMD curl -H "X-Api-Key: $EDC_API_AUTH_KEY" --fail http://localhost:8181/api/check/health
```

Comment:
Please fix the Health-Check

Reply:
I have moved the health check under the management context, however the warnings about an outdated health URL continue to come up there. I assume this is a buggy implementation, because after all we are now using the API endpoint of the management context as the message says. Would wait for MS9 here.

Management-Context:
`http://localhost:11004/api/v1/management/check/health`
instead of the old
`http://localhost:8181/api/check/health`

Reply:
Just as note: here's the original EDC-Connector PR, which changed the endpoints:
https://github.com/eclipse-edc/Connector/pull/2217/files